core-web/libs/block-editor/src/lib/extensions/ai-image-prompt/ai-image-prompt.component.html

```diff
@@ -1,118 +1,32 @@
-                        fill="#7042F0" />
-                    <path
-                        d="M35.9026 8.29011L39.0994 7.31284C39.2522 7.26623 39.3864 7.16951 39.4819 7.03716C39.5773 6.9048 39.6289 6.74394 39.6289 6.57862C39.6289 6.41331 39.5773 6.25244 39.4819 6.12008C39.3864 5.98773 39.2522 5.89101 39.0994 5.8444L35.9026 4.8679L34.9609 1.54959C34.916 1.39096 34.8228 1.25169 34.6953 1.15262C34.5677 1.05354 34.4127 1 34.2534 1C34.0941 1 33.9391 1.05354 33.8115 1.15262C33.684 1.25169 33.5908 1.39096 33.5459 1.54959L32.6049 4.8679L29.4075 5.8444C29.2546 5.891 29.1204 5.98771 29.025 6.12006C28.9295 6.25241 28.8779 6.41329 28.8779 6.57862C28.8779 6.74395 28.9295 6.90483 29.025 7.03718C29.1204 7.16953 29.2546 7.26624 29.4075 7.31284L32.6049 8.29011L33.5459 11.6076C33.5908 11.7663 33.684 11.9056 33.8115 12.0046C33.9391 12.1037 34.0941 12.1572 34.2534 12.1572C34.4127 12.1572 34.5677 12.1037 34.6953 12.0046C34.8228 11.9056 34.916 11.7663 34.9609 11.6076L35.9026 8.29011Z"
-                        fill="#7042F0" />
-                    <path
-                        d="M19.6845 9.67937L16.459 11.0447C16.3233 11.1021 16.2072 11.2002 16.1254 11.3265C16.0437 11.4527 16 11.6014 16 11.7535C16 11.9056 16.0437 12.0542 16.1254 12.1805C16.2072 12.3067 16.3233 12.4048 16.459 12.4623L19.6845 13.8276L21.0001 17.175C21.0555 17.3159 21.1501 17.4364 21.2717 17.5212C21.3934 17.606 21.5366 17.6514 21.6832 17.6514C21.8297 17.6514 21.973 17.606 22.0946 17.5212C22.2163 17.4364 22.3108 17.3159 22.3662 17.175L23.6818 13.8276L26.9067 12.4623C27.0424 12.4048 27.1585 12.3067 27.2402 12.1805C27.322 12.0542 27.3656 11.9056 27.3656 11.7535C27.3656 11.6014 27.322 11.4527 27.2402 11.3265C27.1585 11.2002 27.0424 11.1021 26.9067 11.0447L23.6818 9.67937L22.3662 6.33266C22.3108 6.19183 22.2163 6.07132 22.0946 5.98648C21.973 5.90165 21.8297 5.85635 21.6832 5.85635C21.5366 5.85635 21.3934 5.90165 21.2717 5.98648C21.1501 6.07132 21.0555 6.19183 21.0001 6.33266L19.6845 9.67937Z"
-                        fill="#7042F0" />
-                </svg>
-            </div>
-            <span>
-                <span
-                    ><strong>Generate</strong> an AI image based on your input and requests.
-                </span>
+<ng-container *ngIf="vm$ | async as vm">
+    <p-dialog
+        [(visible)]="vm.showDialog"
```

Comment:
I'm curious if this is not causing unexpected behavior. 

This is a `Two-way binding` and we are passing here a value from the store. Isn't this changing the value of the store?  I know we are using the `onHide` method but that one is called after this value changes to `false`